src/chainstate/burn/operations/leader_block_commit.rs

```diff
@@ -439,6 +451,16 @@ pub struct MissedBlockCommit {
     pub intended_sortition: SortitionId,
 }
 
+impl MissedBlockCommit {
+    pub fn spent_txid(&self) -> Txid {
+        self.input.0.clone()
```

Comment:
Same as above -- can this just return `&Txid`?

Reply:
Fixed in 9c6035c